app/src/main/res/values/strings.xml

```diff
@@ -10,4 +10,5 @@
     <string name="tripBooked">%1$d trip(s) has(have) been booked 🚀🚀</string>
     <string name="tripCancelled">A trip has been cancelled</string>
     <string name="subscriptionError">Oops, something wrong happened</string>
+    <string name="title_activity_main2">MainActivity2</string>
```

Comment:
MainActivity2 is not used

Reply:
Interesting, where would  `MainActivity2` come from?

Reply:
👍

Reply:
No idea where it comes from... Some IDE magic somewhere I guess ✨